### DISCUSSION TITLE
improve panel autosizing

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#48973] [Dashboards, Panels & Visualizations]
**What happened**:

StatPanel: sometimes auto-size is not working

![auto-size](https://user-images.githubusercontent.com/2898361/167122213-a0c443f9-0f1c-4160-9859-463ba63e0838.gif)

![image](https://user-images.githubusercontent.com/2898361/167117596-533ea2be-ab4a-4c3d-beb7-83f62a35f4b3.png)

![image](https://user-images.githubusercontent.com/2898361/167117350-7f9ece0d-e873-46da-89fa-c61af9bb6c63.png)

**What you expected to happen**:

True auto size

this result is reached by adding white space to the text 👇

![image](https://user-images.githubusercontent.com/2898361/167117820-5ee9b53d-3c5e-4ac3-a4e7-93ea4db5b2ad.png)

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

Repet panel, 
Display 23", 

**Environment**: 
- Grafana version: any [also v8.5.1 (61fe4a185)]
- Data source type & version: MSQL
- OS Grafana is installed on: Win10
- User OS & Browser:  Chrome
- Grafana plugins:
- Others:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
